example/ios/AirMapsExplorer/AppDelegate.m

```diff
@@ -11,12 +11,16 @@
 
 #import "RCTRootView.h"
 
+#import <GoogleMaps/GoogleMaps.h>
+
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   NSURL *jsCodeLocation;
 
+  [GMSServices provideAPIKey:@"AIzaSyAeHIC4IG7XKT2Ls5Ti_YZV-6DHQk6dVHE"];
```

Comment:
Pretty sure it's safe. However, this is a key I generated from my personal account. Do you want to use a key from whatever account you used to generate the android key?